### DISCUSSION TITLE
Question: Firely Operation Outcome

### DISCUSSION THREAD

**Discussion by salviashish** [FirelyTeam/firely-net-sdk#1825] [Q&A]
Hi Everyone!

We have implemented a Kafka consumer to post messages to Azure FHIR API using a .Net client using Firely SDK for .Net.

Now the consumer randomly throws FhirOperationOutcome and there aren't any failures on Azure FHIR API. So reaching out for help to understand the cause of FhirOperationOutcome where the OperationOutcome object is null.

Thank you!

**Comment by marcovisserFurore** [comment]
Hi @salviashish,

Can you give us a bit more information? What are the properties of the `FhirOperationException`? So, the status, message and outcome?
